pkg/controller/template/test_data/templates/master/01-master-kubelet/vsphere/units/kubelet.service

```diff
@@ -25,7 +25,7 @@ contents: |
         --client-ca-file=/etc/kubernetes/ca.crt \
         --cloud-provider=vsphere \
         --volume-plugin-dir=/etc/kubernetes/kubelet-plugins/volume/exec \
-        --cloud-config=/etc/kubernetes/cloud.conf \
+         \
```

Comment:
let's remove the line entirely? same in the worker service below

Reply:
currently this is status for kubelet service files in master, and it would require changing the templates to allow skipping this file. Would it be okay to take it as a follow up to update the template to do this for all platforms?

Reply:
EDIT: also I want this PR's scope confined to changing the render function behavior

Reply:
no worries @abhinavdahiya 

we can get this PR in and make whatever small changes as a followup.